### PR TITLE
Remove term fake and create issuer with general names

### PR DESCRIPTION
**Pull request by rafaelbernard**
- Remove fakes and improve names
- Sign for aws server

https://app.asana.com/0/1171124854073033/1193562276340788

### CODE REVIEW

File: description-pbdf.xml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+<SchemeManager version="7">
+	<Id>inz-demo</Id>
+	<Url>https://inz-demo-scheme.s3-ap-southeast-2.amazonaws.com/</Url>
```
Comment (camfindlay) [inline description-pbdf.xml]: do we want to use our irma.nz subdomain for this now?

File: inz-demo/internal-affairs/description.xml [hunk 2]
```diff
@@ -0,0 +1,15 @@
+<Issuer version="4">
+    <ID>internal-affairs</ID>
+    <Name>
+        <en>Demo Internal Affairs Issuer</en>
+        <nl>Demo Internal Affairs Issuer</nl>
+    </Name>
+    <ShortName>
+        <en>Demo Internal Affairs</en>
```
Comment (camfindlay) [inline inz-demo/internal-affairs/description.xml:8]: could be useful to start to line up on the data standards for name, address and a few other things. NZ currently has 3 approved data content standards which would be useful for our PoCs. See https://www.data.govt.nz/manage-data/data-content-standardisation/register-of-government-data-content-requirements/

File: inz-demo/internal-affairs/Issues/name/description.xml [hunk 3]
```diff
@@ -1,14 +1,14 @@
...
+        <en>Demo Internal Affairs full name</en>
+        <nl>Demo Internal Affairs full name</nl>
     </Name>
     <ShortName>
         <en>Full name</en>
         <nl>Name</nl>
     </ShortName>
     <SchemeManager>inz-demo</SchemeManager>
-    <IssuerID>fake-internalaffairs</IssuerID>
+    <IssuerID>internal-affairs</IssuerID>
```
Comment (camfindlay) [inline inz-demo/internal-affairs/Issues/name/description.xml]: I think it's good to keep the internal affairs issuer in as later we might play with adding a new passport credential which also comes from them. Date of birth can also come from internal affairs as they have all the records of birth for NZ.

**Review by camfindlay — APPROVED** [review]
For the immediate need these are fine and we have some options for testing how we change the scheme later once it's in the official IRMA app.

File: description-pbdf.xml (hunk 1 above)
Comment (rafaelbernard) [reply description-pbdf.xml]: I created https://app.asana.com/0/1171124854073033/1193562276340787 for this as we could try some alternatives. I will start that and I can then change this URL.

**Review by ehsan-fj — APPROVED** [review]

File: inz-demo/internal-affairs/description.xml (hunk 2 above)
Comment (rafaelbernard) [reply inz-demo/internal-affairs/description.xml:8]: @camfindlay Sure! I changed to use the standard names. We can improve later creating a full credential with all pieces of the data content standard, but doing now would require lots of changes in all our PoC artefacts. I added the standard document to our library to use it whenever possible.

**Pull request merged**